README.md

```diff
@@ -55,20 +55,52 @@ The project is configured to use gradle. To build the jar, just do:
 
 `gradlew assemble`
 
+or
+
+```
+gradle build
+gradle fatJar
+```
+
 ### To change the code
 
 If you want to open the project, I highly suggest that you use IntelliJ IDEA Community (Free) or Ultimate (paid).
-This IDE is cross platform so it should work on any OS. This project was created using this tool and you can just open 
+This IDE is cross platform so it should work on any OS. This project was created using this tool and you can just open
 it directly. Maybe you can use other tools to open the project but I will not provide any support.
 
 ### To launch
 
 `java -jar swagger-style-validator.jar -s ./path/to/spec.yaml -o ./path/to/options.json`
 
--s|-source : The path to your yaml spec file
--o|-options : (Optional) The path to your json options file 
+or example
```

Comment:
maybe change this to: Another example using the default output path of the jar file